js/widgets/phrasemaker.js

```diff
@@ -267,10 +267,35 @@ class PhraseMaker {
             widgetWindow.destroy();
         };
 
-        widgetWindow.addButton("play-button.svg", PhraseMaker.ICONSIZE, _("Play")).onclick = () => {
+        this._playButton = widgetWindow.addButton("play-button.svg", PhraseMaker.ICONSIZE, _("Play"));
+        
+        this._playButton.onclick = () => {
             logo.turtleDelay = 0;
 
             logo.resetSynth(0);
+            if(this.playingNow) {
+                this._playButton.innerHTML =
+                    '&nbsp;&nbsp;<img src="header-icons/play-button.svg" title="' +
+                    _("Play") +
+                    '" alt="' +
+                    _("Play") +
+                    '" height="' +
+                    PhraseMaker.ICONSIZE +
+                    '" width="' +
+                    PhraseMaker.ICONSIZE +
+                    '" vertical-align="middle">&nbsp;&nbsp;';
```

Comment:
@meganindya did we decide on formatter? looking funny.

Reply:
This is ok for when characters per line exceeds - standard JS convention. We decided on 100 characters per line.